src/main/java/org/jenkinsci/plugins/rundeck/RundeckNotifier.java

```diff
@@ -562,9 +562,9 @@ public synchronized void load() {
             initializeRundeckJobCache();
         }
 
-        public RundeckInstanceBuilder getRundeckBuilder() {
-            return rundeckBuilder;
-        }
+        // public RundeckInstanceBuilder getRundeckBuilder() {
```

Comment:
do we need this? can we remove it?

Reply:
Actually I did try to remove this, a lot of change is needed to remove this. When I was trying so many(10s) test cases also started failing and I decided not to touch this legacy..

Reply:
and getInstanceBuilder method is being used at a couple of place, I think in test cases.